<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputFEN</class>
 <widget class="QWidget" name="InputFEN">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPlainTextEdit" name="FEN_String">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>351</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="InputFEN_2">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>149</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入棋局的FEN字符串：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
